fdm-app/app/components/blocks/nutrient-advice/nutrients.ts

```diff
@@ -0,0 +1,119 @@
+import type { NutrientDescription } from "./types"
+
+export function getNutrientsDescription(): NutrientDescription[] {
+    const nutrientDescription: NutrientDescription[] = [
+        {
+            name: "Stikstof, werkzaam",
+            symbol: "N",
+            type: "primary",
+            unit: "kg N/ha",
+            adviceParameter: "d_n_req",
+            doseParameter: "p_dose_nw",
+        },
+        {
+            name: "Fosfaat",
+            symbol: "P",
+            type: "primary",
+            unit: "kg P2O5/ha",
+            adviceParameter: "d_p_req",
+            doseParameter: "p_dose_p",
+        },
+        {
+            name: "Kalium",
+            symbol: "K",
+            type: "primary",
+            unit: "kg K2O/ha",
+            adviceParameter: "d_k_req",
+            doseParameter: "p_dose_k",
+        },
+        {
+            name: "Organische koolstof",
+            symbol: "EOC",
+            type: "secondary",
+            unit: "kg EOC/ha",
+            adviceParameter: "d_c_req",
+            doseParameter: "p_dose_eoc",
+        },
+        {
+            name: "Zwavel",
+            symbol: "S",
+            type: "secondary",
+            unit: "kg S/ha",
+            adviceParameter: "d_s_req",
+            doseParameter: "p_dose_s",
+        },
+        {
+            name: "Magnesium",
+            symbol: "Mg",
+            type: "secondary",
+            unit: "kg MgO/ha",
+            adviceParameter: "d_mg_req",
+            doseParameter: "p_dose_mg",
+        },
+        {
+            name: "Calcium",
+            symbol: "Ca",
+            type: "secondary",
+            unit: "kg Ca/ha",
+            adviceParameter: "d_ca_req",
```

Comment:
kg CaO/ha to be consistent

Reply:
Resolved in 8b7ea60a64a2b60d866dd134337f5e13e6e76440